spec/jobs/update_user_assessment_attempt_from_class_marker_job_spec.rb

```diff
@@ -12,6 +12,7 @@
   let(:assessment_attempt) { create(:assessment_attempt, user_id: user.id, assessment_id: assessment.id) }
   let(:programme_complete_counter) { create(:programme_complete_counter, programme_id: programme.id) }
   let(:passing_json_body) { File.read("spec/support/class_marker/passing_webhook.json") }
+  let(:timed_out_json_body) { File.read("spec/support/class_marker/timed_out_webhook.json") }
```

Comment:
I assume this file no longer exists? If so we can remove this.

Reply:
Good spot Mike, I've removed this :)